### ISSUE TITLE
AlexNet issue Batch Normalization fc6/bn

### ISSUE THREAD

**Issue by Wronskia** [NVIDIA/caffe#423]
Hello,

I am trying to finetune AlexNet. I get the following error which i don't get when running the same process in caffe :

I0929 15:28:49.480206 19322 net.cpp:184] Created Layer fc6/bn (22)
I0929 15:28:49.480216 19322 net.cpp:561] fc6/bn <- fc6
I0929 15:28:49.480231 19322 net.cpp:530] fc6/bn -> fc6/bn
F0929 15:28:49.480550 19322 blob.hpp:252] Check failed: axis_index < num_axes() (2 vs. 2) axis 2 out of range for 2-D Blob with shape 100 4096 (409600)
*** Check failure stack trace: ***
@ 0x7f8d746ac5cd google::LogMessage::Fail()
@ 0x7f8d746ae433 google::LogMessage::SendToLog()
@ 0x7f8d746ac15b google::LogMessage::Flush()
@ 0x7f8d746aee1e google::LogMessageFatal::~LogMessageFatal()
@ 0x417f58 caffe::Blob::CanonicalAxisIndex()
@ 0x7f8d75519cce caffe::BatchNormLayer<>::LayerSetUp()
@ 0x7f8d756e2f29 caffe::Net::Init()
@ 0x7f8d756e492e caffe::Net::Net()
@ 0x7f8d752b3210 caffe::Solver::InitTrainNet()
@ 0x7f8d752b374d caffe::Solver::Init()
@ 0x7f8d752b3c22 caffe::Solver::Solver()
@ 0x7f8d752c0126 caffe::Creator_SGDSolver()
@ 0x41bce6 caffe::SolverRegistry::CreateSolver()
@ 0x4133e2 train()
@ 0x40deb8 main
@ 0x7f8d72de0830 __libc_start_main
@ 0x40eaf9 _start
@ (nil) (unknown)
root@6409698f2bc4:/workspace/imagenet-finetuning/Alexnet#

Thank you,
Best

**Comment by drnikolaev** [comment]
hi @Wronskia 
AlexNet with BN? Are you using https://github.com/NVIDIA/caffe/tree/caffe-0.16/models/alexnet_bn ?

**Comment by Wronskia** [comment]
Hello @drnikolaev 

Actually, I am using this one : https://github.com/cvjena/cnn-models/blob/master/AlexNet_cvgj/train.prototxt

Thank you,
Best

**Comment by drnikolaev** [comment]
Please check v0.16.5 and reopen the issue if the problem still exists.
